src/server/game/Instances/InstanceScript.h

```diff
@@ -21,6 +21,7 @@
 
 #include "ZoneScript.h"
 #include "Common.h"
+#include "ObjectMgr.h"
```

Comment:
Or no, that won't work either (type itself is defined in `ObjectMgr.h`). I'll think of something.